src/Normalizer/EntityReferenceItemNormalizer.php

```diff
@@ -106,6 +106,10 @@ public function normalize($field_item, $format = NULL, array $context = array())
       $entity = $field_item->getEntity();
       $field_uri = $this->linkManager->getRelationUri($entity->getEntityTypeId(), $entity->bundle(), $field_name, $context);
```

Comment:
I'm going to delete these lines. No point in calling something if we don't need to.